contents/blog/activation-query.mdx

```diff
@@ -0,0 +1,265 @@
+
+### What is PostHog's activation metric?
+
+We can look at session replay. Based on our analysis, users who:
+
+- Watch 5 session replays
+- Set replay filters > 0 times
+
+are much more likely to still be using session replay in 3 months time than people who didn't do those things.
+
+## What do I do with the activation metric?
+
+You push for it, of course! If you know you need people to do actions D, K, and L, you can email them about those things, track it in your metrics, run experiments to get more people using it, etc.
+
+For session replay, we email people pretty soon off the bat if they've shown interest in the product but haven't activated.
+
+## How do I find my activation metric?
+
+Fortunately, by using the query in this article, finding your activation metric should be relatively easy.
+
+Step 1: Create a list of events that you think might "hook" your user into using your product.
+
+Step 2: Create groups of 3-5 events that you can test together. Aim for 5-10 different groupings, including the number of times the event occurs - sometimes users have to do something more than once to reach their "ah-ha" moment.
+
+Step 3: Use the query below to test the metric for retention. You're looking for a retention percentage that's nominally higher than your average retention rate for all users.
+
+- Make sure there are enough users in each month cohort to generate good results. If only 2 groups did all the activation steps, their retention rate isn't going to be very trustworthy.
+
+Remember: retention is your guide. Your activation metric is a reflection of your retention rate - because people who activate are more likely to continue using your product.
+
+### Activation query
+
+If you use PostHog, you can use our same exact query to explore your data and find your activation metric.
+
+First I'll break the query down into bits to explain what's happening, then I'll provide the whole thing in one chunk at the end.
+
+#### Step 1. Get all the orgs who started with your product, broken down by month
+
+Find an event that represents when a user starts with your product. For many products, this can simply be when the user first signed up. Because this is querying off the events table directly, you can't use an Action for this (or any of the events in the query) - it has to be a "manually" collected event (ie one that is defined in your codebase).
+
+Use that event in this step of the query, which then breaks the users apart by month that they started. If you use groups, you'll also need to include the group ID so that you're filtering everything else down the funnel by group ID.
```

Comment:
This query assumes people use groups, I don't have one for people who don't use groups..... it's prob easy to do but I wonder if it's worth doing?